glide.yaml

```diff
@@ -9,6 +9,7 @@ import:
   subpackages:
   - assert
   - require    
+- package: github.com/willf/bitset
```

Comment:
nit: we should pin all packages to a version in glide.yaml to avoid updating other dependencies unless being explicit.